.editorconfig

```diff
@@ -0,0 +1 @@
+../../.editorconfig
```

Comment:
This doesn't appear to be a valid config file for editorconfig. We should probably remove it